test/unit/host_test.rb

```diff
@@ -598,13 +598,13 @@ def teardown
 
   context "location or organizations are not enabled" do
     before do
-      SETTINGS[:locations_enabled] = false
-      SETTINGS[:organizations_enabled] = false
+      @original_loc, SETTINGS[:locations_enabled] = SETTINGS[:locations_enabled], false
```

Comment:
I don't know how taxonomies settings change tomorrow for tests, this keeps tests less fragile, is there a reason to not? without keeping original value we could enable them even if they were disabled before this context